metacat-schema-converters/src/main/java/com/netflix/metacat/shema/converters/HiveMetacatConvertersImpl.java

```diff
@@ -0,0 +1,346 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.netflix.metacat.shema.converters;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Splitter;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+import com.netflix.metacat.common.QualifiedName;
+import com.netflix.metacat.common.dto.AuditDto;
+import com.netflix.metacat.common.dto.DatabaseDto;
+import com.netflix.metacat.common.dto.FieldDto;
+import com.netflix.metacat.common.dto.PartitionDto;
+import com.netflix.metacat.common.dto.StorageDto;
+import com.netflix.metacat.common.dto.TableDto;
+import org.apache.hadoop.hive.metastore.Warehouse;
+import org.apache.hadoop.hive.metastore.api.Database;
+import org.apache.hadoop.hive.metastore.api.FieldSchema;
+import org.apache.hadoop.hive.metastore.api.MetaException;
+import org.apache.hadoop.hive.metastore.api.Partition;
+import org.apache.hadoop.hive.metastore.api.SerDeInfo;
+import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
+import org.apache.hadoop.hive.metastore.api.Table;
+
+import java.time.Instant;
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Collections;
+import java.util.stream.Collectors;
+
+/**
+ * Class implements hiveMetacatConverter interface.
+ */
+public class HiveMetacatConvertersImpl implements HiveMetacatConverters {
+    private static final Splitter SLASH_SPLITTER = Splitter.on('/');
+    private static final Splitter EQUAL_SPLITTER = Splitter.on('=').limit(2);
+
+    @VisibleForTesting
+    Integer dateToEpochSeconds(final Date date) {
```

Comment:
If this is package private why do you need `@VisableForTesting`? Should be accessible from the test class cause it's in the same package.